CHANGELOG.md

```diff
@@ -43,6 +43,9 @@
 * `console.*()` calls in tests are now always intercepted by default. To show them use `--nocapture`. When shown they are always printed in-place instead of after test results, analogous to `cargo test`.
   [#4356](https://github.com/rustwasm/wasm-bindgen/pull/4356)
 
+* Deprecation warning when using an async function as a constructor
+  [#4402](https://github.com/rustwasm/wasm-bindgen/pull/4402)
```

Comment:
One last change: the PR needs a rebase and the new changelog entry be moved to a new "Unreleased" section.